mmv1/third_party/terraform/services/filestore/resource_filestore_backup_test.go

```diff
@@ -113,3 +139,41 @@ resource "google_filestore_backup" "backup" {
 
 `, instName, bkupName)
 }
+
+func testAccFilestoreBackupTags(instName string, bkupName string, tags map[string]string) string {
+
+	return fmt.Sprintf(`
+	resource "google_filestore_instance" "instance" {
+          name     = "%s"
+          location = "us-central1-b"
+          tier     = "BASIC_HDD"
+
+            file_shares {
+              capacity_gb = 1024
+              name        = "share1"
+            }
+
+            networks {
+              network      = "default"
+              modes        = ["MODE_IPV4"]
+              connect_mode = "DIRECT_PEERING"
+            }
+        }
+
+        resource "google_filestore_backup" "backup" {
+          name              = "%s"
+          location          = "us-central1"
+          description       = "This is a filestore backup for the test instance"
+          source_instance   = google_filestore_instance.instance.id
+          source_file_share = "share1"
+
+          labels = {
+            "files":"label1",
+            "other-label": "label2"
+          }
```

Comment:
Do not hard code tags in here, use the parameter provided. Example: https://screenshot.googleplex.com/BmcLcKPfbbjhbji

Reply:
done